aeron-driver/src/main/c/media/aeron_udp_channel_transport.c

```diff
@@ -275,10 +290,17 @@ int aeron_udp_channel_transport_init(
 
     if (aeron_set_socket_non_blocking(transport->fd) < 0)
     {
-        AERON_APPEND_ERR("%", "");
+        AERON_APPEND_ERR("%", "failed to set transport->fd to be non-blocking");
```

Comment:
Should this be `transport->recv_fd`?

Reply:
That looks okay to me.  It sets non-blocking and reports errors on recv_fd a few lines down from that code.